### DISCUSSION TITLE
"Enabled" Cocktails tauchen nicht im Maker Tab auf

### DISCUSSION THREAD

**Discussion by GasgasEC1979** [AndreWohnsland/CocktailBerry#150] [Need help] [answered]
Hallo,
ich habe aktuell das Problem, dass Cocktails, welche ich im Rezepte Tab auf "Enabled" setze, nicht im Maker Tab auftauchen. Andersrum funktioniert es, heißt wenn ich im Rezepte Tab die Haken bei "Enabled" entferne, verschwinden die Cocktails im Maker Tab.
Bei Programmstart tauchen im Maker Tab immer nur die Cocktails auf, welche keine manuell hinzuzufügenden Zutaten benötigen. 
Dieses Problem konnte ich bei der vorherigen Software Version (Version kenne ich nicht genau, ca. 1.5 Jahre alt) dadurch beheben, dass ich bei jeden Cocktail mit manuell hinzuzufügenden Zutaten einmal auf den "Ändern" Button gedrückt habe. Dann war er wieder im "Maker" Tab vorhanden. 
Nach Umstellung auf die aktuellste Software Version, funktioniert dieser "Trick" allerdings nicht mehr. 
Mache ich irgendetwas falsch?

**Comment by AndreWohnsland** [comment]
Hi, 
Generell ist die Programmlogik so, dass im maker nur Rezepte angezeigt werden wo alle Zutaten zusammen an der Maschine angeschlossen oder bei der verfügbar Konfiguration als verfügbar gesetzt sind.
Beispiel: Cocktail 1 braucht Zutat a, b und c, Cocktail 2 braucht a, c und d.
Jetzt ist im Programm a und b an der Maschine angeschlossen und c als verfügbar (steht per Hand hinzufügen neben der Maschine) in der Konfiguration gesetzt. Somit wird Cocktail 1 im maker angezeigt, Maschine macht a und b selbst und fordert den Nutzer auf c per Hand hinzuzufügen. Cocktail 2 wird nicht angezeigt, da Zutat d weder an der Maschine noch als verfügbar deklariert ist.
Früher hat man explizit im Rezept definiert, was per Hand und was per Maschine sein soll, seid längerem berechnet die Software das selbst auf Basis der Konfiguration aus, was es deutlich flexibler macht.
Falls es noch Probleme gibt, kannst du mir ja mal Bilder von Flaschen / verfügbar Konfiguration schicken beziehungsweise dem Cocktail der nicht angezeigt wird.

**Reply by GasgasEC1979** [reply, marked as answer]
Hi,
erstmal vielen Dank für die schnelle Antwort... 
Wie so oft ein Bedienerproblem ;-). Nachdem ich die "verfügbar" Konfiguration angepasst habe, funktioniert alles wie es soll, die Cocktails tauchen dementsprechend auch im Maker Tab auf.
Nur noch eine kurze Frage für mein Verständnis, benötige ich dann überhaupt noch die Funktion "nur selbst hinzufügen" im Zutaten Tab? Im Prinzip macht die Software die Auswahl ja über die "verfügbar" Konfiguration und die Pumpenzuordnung selbstständig.

**Reply by AndreWohnsland** [reply]
Hi, kein Problem.
Die Funktion ist dahingehend nützlich für Zutaten, die du niemals an der Maschine anschließen willst. Mache ich zum Beispiel bei allen Sirups. Somit wird die Zutat auch nicht im Flaschen / bottles tab bei der Auswahl in den Dropdowns angezeigt, wenn du einstellst welche Zutaten an der Machine angeschlossen sind. Dann musst du weniger scrollen, was mit mehr Zutaten angenehm ist.
